deno.json

```diff
@@ -1,11 +1,11 @@
 {
   "tasks": {
     "chel": "deno task lint && deno task build && deno run --allow-net --allow-read=. --allow-write=. build/main.js",
-    "lint": "npx eslint ./src ./scripts",
+    "lint": "deno run --allow-env --allow-read=. --allow-sys --deny-sys=\"networkInterfaces\" npm:eslint@9.31.0 ./src ./scripts",
     "compile": "deno run --allow-run --allow-read=. --allow-write=./dist scripts/compile.ts",
-    "build": "deno run --node-modules-dir --allow-run --allow-read --allow-env --allow-write=./build --allow-net scripts/build.ts",
+    "build": "deno run --node-modules-dir --allow-run --allow-read=. --allow-env --allow-write=./build --allow-net scripts/build.ts",
     "dist": "deno lint . && deno task lint && deno task build && deno task compile",
-    "test": "deno task lint && deno test --allow-read --allow-write --allow-env --allow-ffi --allow-net"
+    "test": "deno task lint && deno test --allow-read=.,$HOME/.cache --allow-write=.,$HOME/.cache --allow-env --allow-ffi --allow-net"
```

Comment:
Getting `throw new Error("Failed to load SQLite3 Dynamic Library", { cause: e });` on `deno task test`. It says `Caused by: NotCapable: Requires read access to "/deno/plug", run again with the --allow-read flag`